src/test/java/util/xcmailr/XcMailrRetrieveEmailsTest.java

```diff
@@ -107,11 +93,11 @@ public void fetchLastEmailByHtmlContent()
     public void fetchEmailsFromTempEmail()
     {
         SendEmail.send(emailAccount, tempEmail, subject, textToSend);
-
+        Selenide.sleep(1000);
```

Comment:
because sometimes the second email doesn't manage to reach the destination. So the tests assert, that there is only one income e-mail, that matches these parameters. In other test methods `fetchEmails` takes waiting over, because it implicitly waits for an e-mail with the requested parameters to come but in this test are two identical e-mails awaited so the built in wait doesn't help.